src/mmw/apps/home/views.py

```diff
@@ -116,6 +116,10 @@ def set_url(layer):
 
 
 def get_client_settings(request):
+    # BiG-CZ mode applies when either request host contains predefined host, or
+    # ?bigcz=true query parameter is present. This covers staging sites, etc.
+    bigcz = (settings.BIGCZ_HOST in request.META.get('HTTP_HOST') or
```

Comment:
There's also [request.get_host](https://docs.djangoproject.com/en/1.11/ref/request-response/#django.http.HttpRequest.get_host) which may be a little more robust.